website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,543 @@
+
+1. Make a `GET` call to
+   [`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+   with the new rule name to verify the login configuration.
+
+   For example:
+
+    <CodeBlockConfig hideClipboard>
+
+    ```shell-session
+    $ curl                                               \
+        --request GET                                    \
+        --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+        ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+    ```
+
+    </CodeBlockConfig>
+    <br/>
+    <CodeBlockConfig hideClipboard>
+
+    ```json
+    {
+      "data": {
+        "backup_auth_types": [
+          "ldap",
+          "token"
+        ],
+        "default_auth_type": "userpass",
+        "disable_inheritance": false,
+        "namespace_path": "dev/"
+      },
+    }
+    ```
+
+    </CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+
+## Delete a login rule
+
+<Tabs>
+<Tab heading="CLI" group="cli">
+
+Use `vault delete` with the 
+[`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+endpoint and rule name to delete a login setting for a namespace. 
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/<unique_rule_name>
+```
+
+For example:
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ vault delete sys/config/ui/login/default-auth/dev-gui-login
+```
+
+</CodeBlockConfig>
+
+</Tab>
+<Tab heading="GUI" group="gui">
+
+You can delete a login rule in two places from the **UI Login Settings** page.
+
+**From the additional options menu for the customization**:
+
+1. Find the login rule you want to delete.
+1. Click the additional option button (three dots).
+1. Select **Delete** from the dropdown menu to bring up the delete confirmation modal.
+
+**From the customization details page**:
+
+1. Find the login rule you want to delete.
+1. Select it to open the login settings details page.
+1. Click the **Delete Rule** button to bring up the delete confirmation modal.
+
+</Tab>
+<Tab heading="API" group="api">
+
+Make a `DELETE` call to the
+[`sys/config/ui/login/default-auth/{rule_name}`](/vault/api-docs/system/config-ui-login-default-auth)
+endpoint to delete a login setting for a namespace. 
+
+```shell-session
+$ curl                                             \
+  --request DELETE                                 \
+  --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+  http://127.0.0.1:8200/v1/sys/config/ui/login/default-auth/<unique_rule_name>| jq
+```
+
+For example, to delete the rule "dev-gui-login":
+
+<CodeBlockConfig hideClipboard>
+
+```shell-session
+$ curl                                               \
+    --request DELETE                                 \
+    --header "X-Vault-Token: ${VAULT_TOKEN}"         \
+    ${VAULT_ADDR}/v1/sys/config/ui/login/default-auth/dev-gui-login | jq
+```
+
+</CodeBlockConfig>
+
+</Tab>
+</Tabs>
+
+## Using a Direct Link
```

Comment:
I wasn't really sure if this should live here. One thought was honoring the Next steps pattern and creating a new route/docs for "Vault GUI query params" because I'd also like to document the behavior of using the `wrapped_token` query parameter somewhere

Reply:
I like having it in one view, but I can also see the benefit of breaking it out for more detail. I'll defer to whatever the docs folks suggest.

Reply:
I think maybe this should be sentenced cased? `Using a direct link`